### DISCUSSION TITLE
How to deteremine if the ffmpeg faild when using Process component ?

### DISCUSSION THREAD

**Discussion by foremtehan** [symfony/symfony#47443] [Q&A] [answered]
Wondering, How can i determine ffmpeg failed when using symfony Process component? ffmpeg failed but i won't get any exception

**Comment by stof** [comment, marked as answer]
Check the exit code of the process. that's how failure are reported on the CLI. And then, check the output for details.

There is also a `mustRun()` method on the Process class that you can use instead of `run()`, that does such checks automatically.
